src/helpers/scrollAnimationProvider.ts

```diff
@@ -0,0 +1,20 @@
+import useBreakpoints from 'hooks/useBreakpoints'
+import useScrollPercent from 'hooks/useScrollPercent'
+
+export default function (animationName: string) {
+  const scroll = useScrollPercent()
+  const { onlyPhones, tablet } = useBreakpoints()
+
+  if (onlyPhones && tablet) animationName = animationName + 'Mobile'
```

Comment:
@backmeupplz, I have a few arguments why we can't do this in CSS:
0) We can't use `@media` to define different `@keyframes` for different devices. I tried googling it, here's a [solutions](https://stackoverflow.com/questions/20407906/defining-keyframe-animations-inside-media-queries), which can't be used by us because:
1) We use different precents for mobile and desktop animations. So we can't define `--variables` in `:root`
2) We can go by applying `id` to each element which is animated and use `@media` for each `id` to change the animation name